test.mjs

```diff
@@ -11,7 +10,7 @@ for (let n = 1; n <= 53; n++) {
 
     t.alike(c.decode(bitfield(n), c.encode(bitfield(n), i)), b, i.toString(2))
 
-    t.alike(c.decode(bitfield(n), c.encode(bitfield(n), b)), b, util.inspect(b))
```

Comment:
Bit controversial perhaps but as it doesn't exist on Bare (?) and is only used for failure introspection, I just removed it.